README.md

```diff
@@ -130,7 +154,7 @@ To set up the Firebase Service Account for this project, follow these steps:
 ## 4. Run the API
 
 ```sh
-npm run dev
+npm run dev:startserver
```

Comment:
This needs to be `npm run dev` since that sill install, create typedefs, seed and start the server. The `npm run dev:startserver` is called in the `npm run dev`

Reply:
It did not work when I did that I would get this error
```> findadoc-server@1.0.0 generate
> echo '➡ ➡ ➡ Generating types from graphql schema ➡ ➡ ➡' && graphql-codegen --config ./typesgeneratorconfig.ts

'➡ ➡ ➡ Generating types from graphql schema ➡ ➡ ➡' 
(node:15732) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
(Use `node --trace-deprecation ...` to show where the warning was created)✔ Parse Configuration
⚠ Generate outputs
  ❯ Generate to src/typeDefs/gqlTypes.ts
    ✔ Load GraphQL schemas
    ✔ Load GraphQL documents
    ✖ Cannot use GraphQLObjectType "Contact" from another module or rea…  
Running lifecycle hook "afterStart" scripts...
[CLI] Loading Schemas
[CLI] Loading Documents
[CLI] Generating output
```

Reply:
We figured it out!